tasks/check-example.js

```diff
@@ -0,0 +1,65 @@
+/*eslint-env es6*/
+
+const http       = require('http');
+const path       = require('path');
+const serveFiles = require('serve-files');
+const spawn      = require('child_process').spawn;
```

Comment:
Watch out, we might add a linter rule that prohibits this sort of ASCII art in the future :).